### PR TITLE
Add tooltips to chart options

### PR DESCRIPTION
**Pull request by hillsn**
added tooltips to chart-options.jsx

changed color of buttons in app.jsx

### CODE REVIEW

File: src/client/js/chart-options.jsx [hunk 1]
```diff
@@ -73,6 +75,10 @@ class ChartOptions extends React.Component {
                   {topicMenuItems}
                 </Select>
               </FormControl>
+              <Tooltip title="Select the meter you would like to receive data from. You may
+                select multiple to overlap data.">
+	            <InfoIcon />
+	          </Tooltip>
```
Comment (kjiwa) [inline src/client/js/chart-options.jsx]: Can you fix the indentation of this? Every new level should be indented by 2 extra spaces. Also, put the Tooltip title text all on one line. It should look like this:

<Tooltip title="Select the meter you would like to receive data from. You may select multiple to overlap data.">
  <InfoIcon />
</Tooltip>

File: src/client/js/chart-options.jsx [hunk 2]
```diff
@@ -83,6 +89,9 @@ class ChartOptions extends React.Component {
                   onChange={this.handleStartDateTimeChange.bind(this)}
                   InputLabelProps={{ shrink: true }} />
               </FormControl>
+			  <Tooltip title="Enter the date and time you would like the sample to start.">
```
Comment (kjiwa) [inline src/client/js/chart-options.jsx]: Same here. Fix this indentation. Uses spaces only, no tabs.

Comment (hillsn) [reply src/client/js/chart-options.jsx]: Should be correct now, still a little confused on proper formatting.

File: src/client/js/chart-options.jsx [hunk 3]
```diff
@@ -73,6 +75,10 @@ class ChartOptions extends React.Component {
                   {topicMenuItems}
                 </Select>
               </FormControl>
+              <Tooltip title="Select the meter you would like to receive data from. You may
+                select multiple to overlap data.">
+                  <InfoIcon />
+	            </Tooltip>
```
Comment (kjiwa) [inline src/client/js/chart-options.jsx]: The indentation is still off. </Tooltip> should be "unindented"

**Pull request merged**